src/mscorlib/src/System/Threading/SynchronizationContext.cs

```diff
@@ -108,7 +108,7 @@ public virtual void Send(SendOrPostCallback d, Object state)
 
         public virtual void Post(SendOrPostCallback d, Object state)
         {
-            ThreadPool.QueueUserWorkItem(new WaitCallback(d), state);
+            ThreadPool.QueueUserWorkItem(new WaitCallback(d), state, preferLocal: true);
```

Comment:
As with above, this is a viewable behavioral change.  How do you know it's not negatively impacted some scenario?